### PR TITLE
Udate ReadMe - Silly and Fun

### PR DESCRIPTION
**Pull request by brandondombrowsky**
In addition to what we talked about, I think a ReadMe should be fun--get the prospective user excited! This is my pitch at cutting back, while saying more, and keeping it light.

### CODE REVIEW

**Comment by alyoshenka** [comment]
Remember to assign reviewers so we get notified that you want a review.

**Comment by alyoshenka** [comment]
Also if you add it to the project it will show up on the board.

File: README.md [hunk 1]
```diff
@@ -1,66 +1,39 @@
...
-
-## Useful Links
-This software is used with lights
-- something.com
-This software is used with plugs
-- somethingelse.com
-This software is used with vents
-- somethingmore.com
+## Cloud vs PI-Hole
+- blah-blah-blah
```
Comment (alyoshenka) [inline README.md:22]: Personally I'm against placeholders in the front-facing readme, but this is ok

**Review by alyoshenka — APPROVED** [review]
Looks good. Short and sweet.

**Review by itsjpb — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (brandondombrowsky) [reply README.md:22]: The placeholder is only temporary

**Pull request merged**